latch/registry/utils.py

```diff
@@ -0,0 +1,355 @@
+import json
+import keyword
+import re
+from dataclasses import fields, is_dataclass
+from enum import Enum, EnumMeta
+from typing import (
+    Annotated,
+    Dict,
+    List,
+    Optional,
+    Type,
+    TypeVar,
+    Union,
+    get_args,
+    get_origin,
+)
+
+import latch.gql as gql
+from latch.types import LatchDir, LatchFile
+
+T = TypeVar("T")
+
+account_root_regex = re.compile("^account_root\/(?P<root>[^/]+)(?P<path>\/.*)$")
+mount_regex = re.compile("^mount\/(?P<mount>[^/]+)(?P<path>\/.*)$")
+
+shared_regex = re.compile("^shared(?P<path>\/.*)$")
+
+
+def clean(name: str):
+    # https://stackoverflow.com/a/3305731
+    cleaned = re.sub("\W|^(?=\d)", "_", name)
+
+    if keyword.iskeyword(cleaned):
+        cleaned = f"_{cleaned}"
+    return cleaned
+
+
+def to_python_type(
+    registry_type: Dict,
```

Comment:
```suggestion
    registry_type: Dict[str, object],
```

Reply:
shouldn't it be `Dict[str, Dict]`?

Reply:
`Dict[str, Dict]` is the same as `Dict[str, Dict[Any, Any]]` and we do not allow `Any`